backend/src/test/java/com/skr/virtuallibrary/controllers/AuthorControllerTest.java

```diff
@@ -0,0 +1,110 @@
+package com.skr.virtuallibrary.controllers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.skr.virtuallibrary.auth.JwtService;
+import com.skr.virtuallibrary.dto.AuthorDto;
+import com.skr.virtuallibrary.services.AuthorService;
+import org.assertj.core.api.Assertions;
+import org.instancio.Instancio;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@AutoConfigureMockMvc(addFilters = false)
+@WebMvcTest(AuthorController.class)
+class AuthorControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private AuthorService authorService;
+
+    @MockBean
+    private JwtService jwtService;
```

Comment:
czy to jest potrzebne? nie widać by było tu używane...

Reply:
Bez tego jest błąd: java.lang.IllegalStateException: Failed to load ApplicationContext for [WebMergedContextConfiguration@23444a36 testClass = com.skr.virtuallibrary.controllers.UserControllerTest, locations = [], ...

Reply:
ok